test/IntraBCTest.cc

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright(c) 2019 Netflix, Inc.
+ * SPDX - License - Identifier: BSD - 2 - Clause - Patent
+ */
+
+/******************************************************************************
+ * @file IntraBCTest.cc
+ *
+ * @brief Unit test of Intra BC:
+ * - av1_is_dv_valid
+ *
+ * @author Cidana-Edmond
+ *
+ ******************************************************************************/
+
+#include "gtest/gtest.h"
+// workaround to eliminate the compiling warning on linux
+// The macro will conflict with definition in gtest.h
+#ifdef __USE_GNU
+#undef __USE_GNU  // defined in EbThreads.h
+#endif
+#ifdef _GNU_SOURCE
+#undef _GNU_SOURCE  // defined in EbThreads.h
+#endif
+#include "EbDefinitions.h"
+#include "EbAdaptiveMotionVectorPrediction.h"
+#include "util.h"
+
+/**
+ * @brief Unit test of Intra BC:
+ * - av1_is_dv_valid
+ *
+ * Test strategy:
+ * sending preset parameter set of DV and get if it is validated
+ *
+ * Expected result:
+ * the result of av1_is_dv_valid should be same as the value in test parameter
+ * set
+ *
+ */
+
+namespace {
+using std::make_tuple;
+
+typedef std::tuple<MV,        /**< DV */
+                   int,       /**< offset of MI row */
+                   int,       /**< offset of MI colum */
+                   BlockSize, /**< mi block size */
+                   int /**< flag of test is validation: 1--true, 0==false */
```

Comment:
This field can be commented as "result", which will be more clear. Since the function is also return 0/1.

Reply:
modified the comment to "the result of dv validation function"